docs/learn/grammar/verbs.mdx

```diff
@@ -92,28 +92,28 @@ In the present tense, the endings for the two conjugations are the following:
 - First conjugation (stems ends in a consonant): **-`u`, -`eš`, -`e`, -`emo`, -`ete`, -`ut`**
 - Second conjugation (`i` conjugation): **-`ju`, -`iš`, -`i`, -`imo`, -`ite`, -`et`**
 
-Instead of **-`u`/-`ju`** in the first person singular, **-`em`/-`im`** can be used as well: :isv[nesem], _`hvalim`_. The former are characteristic especially for East Slavic and Polish, the latter for Slovak and South Slavic.
+Instead of **-`u`/-`ju`** in the first person singular, **-`em`/-`im`** can be used as well: :isv[nesem], :isv[hvalim]. The former are characteristic especially for East Slavic and Polish, the latter for Slovak and South Slavic.
```

Comment:
```suggestion
Instead of `-u`/`-ju` in the first person singular, **-`em`/-`im`** can be used as well: :isv[nesem], :isv[hvalim]. The former are characteristic especially for East Slavic and Polish, the latter for Slovak and South Slavic.
```